docs/sources/administration/configuration.md

```diff
@@ -343,6 +343,12 @@ Defaults to `private`.
 
 <hr />
 
+## [database]
+
+### datasource_limit
+
+The upper limit of number of data sources returned in the response, set this if you need to override the default.
```

Comment:
```suggestion
The upper limit of number of data sources returned in the response. Set this if you need to override the default.
```

Reply:
What is the limit? I do not see it stated anywhere.

Reply:
5000 is the default limit